sdks/python/apache_beam/utils/windowed_value.py

```diff
@@ -270,6 +274,79 @@ def __reduce__(self):
 
 
 # TODO(robertwb): Move this to a static method.
+
+
+class WindowedBatch(object):
```

Comment:
Maybe make this an interface, have this as the first concrete implementation (future implementations may have timestamps stored inside the value, for example).